app/Locale/pt_BR/translations.php

```diff
@@ -1202,134 +1202,133 @@
+    'TRL - Turkish Lira' => 'TRL - Lira turca',
+    'The project email is optional and could be used by several plugins.' => 'O e-mail do projeto é opcional e pode ser usado por diversos plugins.',
+    'The project email must be unique across all projects' => 'O e-mail do projeto precisa ser único entre todos os projetos',
+    'The email configuration has been disabled by the administrator.' => 'A configuração de e-mail foi desabilitada pelo administrador.',
+    'Close this project' => 'Fechar este projeto',
+    'Open this project' => 'Abrir este projeto',
+    'Close a project' => 'Fechar um projeto',
+    'Do you really want to close this project: "%s"?' => 'Deseja realmente fechar este projeto: "%s"?',
+    'Reopen a project' => 'Reabrir um projeto',
+    'Do you really want to reopen this project: "%s"?' => 'Deseja realmente reabrir este projeto: "%s"?',
+    'This project is open' => 'Este projeto está aberto',
+    'This project is closed' => 'Este projeto está fechado',
+    'Unable to upload files, check the permissions of your data folder.' => 'Incapaz de enviar arquivos. Verifique as permissões da sua pasta de dados.',
+    'Another category with the same name exists in this project' => 'Outra categoria com o mesmo nome existe neste projeto',
+    'Comment sent by email successfully.' => 'Comentário enviado por email com sucesso.',
+    'Sent by email to [%s](mailto:%s) (%s)' => 'Enviar por email para [%s](mailto:%s) (%s)',
+    'Unable to read uploaded file.' => 'Incapaz de ler arquivos enviados.',
+    'Database uploaded successfully.' => 'Base de dados enviado com sucesso.',
+    'Task sent by email successfully.' => 'Tarefa enviada por e-mail com sucesso.',
+    'There is no category in this project.' => 'Não há categoria neste projeto.',
+    'Send by email' => 'Enviar por e-mail',
+    'Create and send a comment by email' => 'Criar e enviar um comentário por e-mail',
+    'Subject' => 'Assunto',
+    'Upload the database' => 'Enviar uma base de dados',
+    'You could upload the previously downloaded Sqlite database (Gzip format).' => 'Você pode enviar uma base de dados Sqlite baixada anteriormente (formato Gzip).',
+    'Database file' => 'Arquivo de base de dados',
+    'Upload' => 'Enviar',
+    'Your project must have at least one active swimlane.' => 'Seu projeto precisa ter pelo menos uma raia ativa.',
+    'Project: %s' => 'Projeto: %s',
+    'Automatic action not found: "%s"' => 'Ação automática não encontrada: "%s"',
+    '%d projects' => '%d projetos',
+    '%d project' => '%d projeto',
+    'There is no project.' => 'Não há projeto.',
+    'Sort' => 'Ordenar',
+    'Project ID' => 'ID do projeto',
+    'Project name' => 'Nome do projeto',
+    'Public' => 'Público',
+    'Private' => 'Privado',
+    '%d tasks' => '%d tarefas',
+    '%d task' => 'd% tarefa',
```

Comment:
There is a typo here `d%` instead of `%d` (that prevent automated tests to pass).